api.js

```diff
@@ -562,6 +562,22 @@ module.exports = done => {
     webhooksRoutes(db, server);
     settingsRoutes(db, server, settingsHandler);
 
+    if (process.env.NODE_ENV === "test") {
+        server.get(
+            { name: 'api-methods', path: '/api-methods/:arg' /*, someKey: Joi.object({arg: Joi.string()})*/ },
+            tools.responseWrapper(async (req, res) => {
+                res.charSet('utf-8');
+    
+                // const obj = Object.assign({name: "initial name"}, server.router.getRoutes().getsettingskey.spec);
```

Comment:
Remove unneeded comments, no reason to keep these in the code

Reply:
removed the unneeded comments